rllib/tests/test_io.py

```diff
@@ -146,7 +150,8 @@ def test_agent_input_postprocessing_enabled(self):
                         del data["advantages"]
                         if "value_targets" in data:
                             del data["value_targets"]
-                        out.append(data)
+                        data = _to_json(data, [])
+                        out.append(json.loads(data))
```

Comment:
wait, why do we ``_to_json()`` then ``json.loads()`` immediately after?

Reply:
aha